redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -218,6 +228,8 @@ internal open class UIViewLazyList() : LazyList<UIView>, ChangeListener {
         UICollectionViewScrollDirection.UICollectionViewScrollDirectionHorizontal
       }
     }
+
+    this.isVertical = isVertical
```

Comment:
should we set this at the top of this method? not 100% about this, but if setting `collectionViewFlowLayout.scrollDirection` triggers a redraw we probably want this value to be set ahead of time. we need a horizontal list test/sample

Reply:
Good point, I can move it up